src/modules/expression-evaluator/src/evaluateExpression.test.ts

```diff
@@ -501,13 +501,10 @@ test('Input is an array', () => {
   })
 })
 
-test('Input is malformed JSON string', () => {
-  expect(
-    async () =>
-      await evaluateExpression(
-        '{"operator":"=", "children":[{"value":6},{"operator":"+", "children":[{"value":6},{"value":6}]}]}}'
-      )
-  ).rejects.toThrow('Invalid JSON String')
+test('Input is an string', () => {
+  return evaluateExpression('Friday drinks?').then((result: any) => {
+    expect(result).toEqual('Friday drinks?')
+  })
 })
```

Comment:
I was wondering if there is any tests that can be added to cover the changes on action with output.